### ISSUE TITLE
Featured work / case study

### ISSUE THREAD

**Issue by jenkack** [labopm/website#26]
Sean, do you see having anything for this page by launch, or should we hide for the time being?

https://federalist-proxy.app.cloud.gov/site/labopm/website/what-we-do/featured-work/

**Comment by humancompanion** [comment]
Sean's comment in the Content Word doc was:
"Add Didatics for GSA, VEO, USAJOBS CDC"

I suspect he meant didactics. But this was before Pooja was on board and writing up case studies. I'd talk to her next about the status of that work.

**Comment by jenkack** [comment]
Yep, we'll hear from her in fire swamp and comms meetings, I'll add myself this to this one as well to coordinate with folks as needed to resolve by launch.

**Comment by jenkack** [comment]
Revisit name when we put it up: Case Studies? Projects?

**Comment by jenkack** [comment]
@timtimtimtimofficial 

https://federalist-proxy.app.cloud.gov/preview/labopm/website/content-updates/what-we-do/featured-work/usajobs.html

In addition to any tweaking you're going to do, what about:

- Adding the Contact Us module in sidebar (I tried but couldn't see how it was included on other pages)
- Add screengrab of blueprint (also wasn't sure how to do this, maybe you can give me a walkthrough so I know for next time)
